src/main/scala/EnsimePlugin.scala

```diff
@@ -24,7 +24,7 @@ object EnsimeKeys {
     "Start up the ENSIME server and index your project."
   )
 
-  val ensimeServerVersion = settingKey[String](
+  val ensimeServerVersion = taskKey[String](
```

Comment:
setting is just a performance optimisation, user can still override (and uses exactly the same syntax). The difference is that it is always evaluated, instead of evaluated only once.